src/test/java/net/sf/jabref/model/entry/BibEntryTests.java

```diff
@@ -43,7 +43,7 @@ public void testDefaultConstructor() {
         // we have to use `getType("misc")` in the case of biblatex mode
         Assert.assertEquals("misc", entry.getType());
         Assert.assertNotNull(entry.getId());
-        Assert.assertNull(entry.getField("author"));
+        Assert.assertFalse(entry.getFieldOptional("author").isPresent());
```

Comment:
Better `assertEquals(Optional.empty(), entry.getField(....))` in this way one gets a better failure message. Same applies to the other tests, i.e. `assertEquals(Optional.of("2003-03"), emptyEntry.getFieldOrAlias("date"))` below.